lib/go/test/go.mod

```diff
@@ -1,130 +1,195 @@
-module github.com/onflow/kitty-items/lib/go/test
+module github.com/onflow/nft-storefront/lib/go/test
 
 go 1.18
 
 require (
-	github.com/onflow/cadence v0.29.0-stable-cadence-4
-	github.com/onflow/flow-emulator v0.39.0-stable-cadence-4
-	github.com/onflow/flow-ft/lib/go/templates v0.2.0
-	github.com/onflow/flow-go-sdk v0.29.0-stable-cadence-4
-	github.com/onflow/flow-nft/lib/go/contracts v1.0.1-0.20221110213304-0cc31b1b0607
-	github.com/onflow/flow-nft/lib/go/templates v0.0.0-20221110213304-0cc31b1b0607
-	github.com/stretchr/testify v1.8.0
+	github.com/onflow/cadence v1.0.0-M7
+	github.com/onflow/flow-emulator v1.0.0-M5
+	github.com/onflow/flow-ft/lib/go/contracts v0.7.1-0.20240205224107-320aa3cf09e0
+	github.com/onflow/flow-ft/lib/go/templates v0.7.1-0.20240205224107-320aa3cf09e0
+	github.com/onflow/flow-go-sdk v1.0.0-M4
+	github.com/onflow/flow-nft/lib/go/contracts v1.1.1-0.20240205233530-86ee8c352fa6
```

Comment:
Let's make sure that we're using all the latest versions of these. Some of these versions look pretty old and I wouldn't be surprised if new versions break some things in your contracts. If you just do a `go get github.com/onflow/flow-emulator@v1.0.0-preview.15`, that should get all the latest versions of cadence, emulator, and flow-go-sdk, then you'll need to get the latest commits of the [flow-nft]([7cbc99e](https://github.com/onflow/flow-nft/pull/126/commits/7cbc99efb609eea69295f75b858d9d5158806d48)) and [flow-ft](https://github.com/onflow/flow-ft/pull/131/commits/a9c3316609561d526deb64ef57efb5e526ebb02f) repos as well.